OracleWebLogic/samples/12213-webtier-apache/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Example of Apache Web Server with WebLogic plugin for load balancing WebLogic on Docker Containers
+#
+# Copyright (c) 2016-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# fmw_12.2.1.3.0_wls_Disk1_1of1.zip 
+# Download the Generic installer from http://www.oracle.com/technetwork/middleware/weblogic/downloads/wls-for-dev-1703574.html and save it in this same folder.
```

Comment:
The URL for webtier download is http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html